extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1000,6 +1006,10 @@ export class AmpA4A extends AMP.BaseElement {
 
   /** @override  */
   unlayoutCallback() {
+    if (!isExperimentOn(this.win, 'a4a-fie-unlayout-enabled') &&
```

Comment:
Right this experiment exists as an easy means to disable the new behavior as opposed to enable it.  If this is a concern, I can invert and set to 100% enabled in prod/canary configs